docs/cis_self_assessment.md

```diff
@@ -1653,9 +1601,21 @@ etcd is a highly available key-value store used by Kubernetes deployments for pe
 Authentication to etcd is based on whether the certificate presented was issued by a trusted certificate authority. There is no checking of certificate attributes such as common name or subject alternative name. As such, if any attackers were able to gain access to any certificate issued by the trusted certificate authority, they would be able to gain full access to the etcd database.
 </details>
 
-**Result:** Pass
+**Result:** **Not Applicable**
+
+**Audit**
+Run the below command on the master node.
+
+```bash
+/bin/ps -ef | /bin/grep etcd | /bin/grep -v grep
+```
```

Comment:
Audit is incomplete...you also need to find the apiserver process to get the path to its ca file: 
```
ps -ef | grep apiserver | /bin/grep -v grep
```
That maps to this statement in the verification "...by the `--client-ca-file` for apiserver"